include/picongpu/CMakeLists.txt

```diff
@@ -375,6 +375,8 @@ include_directories(BEFORE ${PIC_EXTENSION_PATH}/include)
 ################################################################################
 
 file(GLOB_RECURSE SRCFILES "*.cpp")
```

Comment:
maybe we just set a "DEVICESRCFILES" variable here so we have it all together (and not scattered)

Reply:
```cmake
# host-only sources
file(GLOB_RECURSE SRCFILES "*.cpp")
# device sources containing kernels
set(DEVICESRCFILES "${CMAKE_CURRENT_SOURCE_DIR}/main.cpp")

# separate host only and accelerator code
list(REMOVE_ITEM SRCFILES ${DEVICESRCFILES})
```

Reply:
note `list(REMOVE_ITEM SRCFILES ${ACCSRCFILES})` is an undocumented feature in cmake